tests/tls_lib_test.sh

```diff
@@ -116,7 +117,7 @@ MODEM_DEVICE = ${TOB_DEVICE}
 MODEM_BAUDRATE = ${TOB_BAUDRATE}
 EOF
 
-${CLOUD_SUPPORT_DIR}/client-openssl/build/client_sample https://localhost:${SERVER_PORT} ${CLIENT_SIGNING_CERT} signing ${SERVER_CA} 2>&1 >${CLIENT_STDOUT} &
```

Comment:
We can now using ToB OpenSSL engine based client certificate read instead of requiring it to be extracted then passed back in.